modules/osdk-internal-crds.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-internal-crds_{context}"]
+= Internal or data-only objects
+
+It is a common practice for Operators to use Custom Resource Definitions (CRDs)
```

Comment:
s/It is a common practice/It is common practice ?
Not 100% sure about this one. _It is common practice_ sounds better to me.